### PR TITLE
APP-1546: Add PSGenerateCodeRequestParameters

### PR DESCRIPTION
**Pull request by eunicegalang**

### CODE REVIEW

File: wallet/src/main/java/com/paysera/lib/wallet/entities/generator/PSGenerateCodeRequestContext.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package com.paysera.lib.wallet.entities.generator;
+
+import com.google.gson.annotations.SerializedName;
+
+public class PSGenerateCodeRequestContext {
```
Comment (gilbertparreno) [inline wallet/src/main/java/com/paysera/lib/wallet/entities/generator/PSGenerateCodeRequestContext.java:5]: I think we could just rename this to be uniform with other classes on this SDK.

File: wallet/src/main/java/com/paysera/lib/wallet/entities/generator/PSGenerateCodeRequestParameters.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package com.paysera.lib.wallet.entities.generator;
+
+import com.google.gson.annotations.SerializedName;
+
+public class PSGenerateCodeRequestParameters {
```
Comment (gilbertparreno) [inline wallet/src/main/java/com/paysera/lib/wallet/entities/generator/PSGenerateCodeRequestParameters.java:5]: same here

**Review by gilbertparreno — CHANGES_REQUESTED** [review]

File: wallet/src/main/java/com/paysera/lib/wallet/entities/generator/PSGenerateCodeRequestContext.java (hunk 1 above)
Comment (eunicegalang) [reply wallet/src/main/java/com/paysera/lib/wallet/entities/generator/PSGenerateCodeRequestContext.java:5]: What do you mean? Moving forward we should add PS prefix for SDK entities, don't we?

Comment (gilbertparreno) [reply wallet/src/main/java/com/paysera/lib/wallet/entities/generator/PSGenerateCodeRequestContext.java:5]: it looks like a alien class here in wallet-sdk :) but its okay if we're moving into this naming convention.

**Pull request merged**